README.md

```diff
@@ -33,3 +33,8 @@ This is a component that animates through an image sprite.
 - startFrame **{number}** - the frame to start animation
 - timeout **{number}** - the amount of millisecond between frame
 - stopLastFrame **{bool}** - stops animation from looping
+
+**Only required for two-dimensional sprites**
+
+- frameCount **{number}** - the total frame count of the sprite
```

Comment:
Ah I see, good call. @dlindenkreuz I'm good with the changes then go ahead and merge. I added you to the repo so you should be able to merge. I'll publish to npm today when I get some stable Internet. 🍻